Source/WebCore/css/CSSStyleSheet.cpp

```diff
@@ -88,6 +91,21 @@ Ref<CSSStyleSheet> CSSStyleSheet::createInline(Ref<StyleSheetContents>&& sheet,
     return adoptRef(*new CSSStyleSheet(WTFMove(sheet), owner, startPosition, true, true));
 }
 
+ExceptionOr<Ref<CSSStyleSheet>> CSSStyleSheet::create(Document& document, Init&& init)
+{
+    URL baseURL;
+    if (init.baseURL.isNull())
+        baseURL = document.baseURL();
+    else {
+        baseURL = document.completeURL(init.baseURL);
+        if (!baseURL.isValid())
+            return Exception { NotAllowedError, "Base URL is invalid"_s };
```

Comment:
`u = new URL("chrome://");` throws in Chrome and Firefox, saying the URL is invalid. However, it doesn't throw in Safari so it seems the behavior difference is in our URL parser (cc @achristensen07). I know we're the only browser matching the URL spec in some cases so this may be one of these cases. We may need to update the test to use a URL that is recognized as invalid in all browsers.